docs/source/deploying/prefixed-root.md

```diff
@@ -0,0 +1,58 @@
+---
+myst:
+  html_meta:
+    "description": "Prefixed (non-root) deployment for Volto"
+    "property=og:description": "Prefixed (non-root) deployment for Volto"
+    "property=og:title": "Prefixed (non-root) deployment"
+    "keywords": "Volto, deployment"
+---
+
+# Prefixed (non-root) deployment
+
+If you're integrating a Volto website within another existing website, you may need to run Volto on a virtual path inside that website instead of the root path.
+
+The first step is to set an environment variable `RAZZLE_PREFIX_PATH` to the prefixed path of your Volto.
+For example, if you want Volto's root to be hosted at `http://example.com/my-prefix`, you need to start Volto with:
+
+```shell
+RAZZLE_PREFIX_PATH=/my-prefix yarn start
+```
+
+If you need to debug and understand how the requests are rewritten by the Volto SSR, you can add the following environment variables to the Volto start line:
+
+
+```shell
+DEBUG_HPM=true DEBUG=superagent RAZZLE_PREFIX_PATH=/my-prefix yarn start
+```
+
+The prefix location will be used regardless of how you start Volto, whether in development or production mode.
+When developing, though, if your backend is something other then `http://localhost:8080`, you'll need to provide your own solution for how to handle things.
+
+For a production setup, when hosting Volto behind a proxy HTTP server, you can configure your rewrite rules to something like the following, in this case for Apache.
+
+```apache
```

Comment:
The recommended deployment setup is using Docker. I know that the documentation about this is in a sorry state, but we cannot add more confusion about this by dropping an `Apache` config here. I don't have a good solution for this one :( 

In the upcoming sprints we should work on documenting properly the deployment story.

/cc @ericof @davisagli @stevepiercy @fredvd